21CB46/HTML-CSS
# Add -  Tribute-Page

An Add the Tribute Page by using HTML-CSS.

ADD #4 

Video -

https://github.com/21CB46/HTML-CSS/assets/91898349/02e92a2e-1183-428b-b3f3-01586eef55b7

How should this be tested?

1. Go to the IDE.
2. Open the Tribute-Page folder.
3. and see the code.
